src/FakeItEasy/Core/AssemblyExtensions.cs

```diff
@@ -14,7 +14,16 @@ public static bool ReferencesFakeItEasy(this Assembly assembly)
         {
             Guard.AgainstNull(assembly, "assembly");
 
+#if FEATURE_REFLECTION_GETASSEMBLIES
             return assembly.GetReferencedAssemblies().Any(r => r.FullName == TypeCatalogue.FakeItEasyAssembly.FullName);
+#else
+            var fakeItEasyLibraryName = TypeCatalogue.FakeItEasyAssembly.GetName().Name;
+            var referencingLibraries = Microsoft.Extensions.PlatformAbstractions.PlatformServices.Default.LibraryManager.GetReferencingLibraries(fakeItEasyLibraryName);
+            return referencingLibraries
+                .SelectMany(info => info.Assemblies)
+                .Select(info => Assembly.Load(new AssemblyName(info.Name)))

```

Comment:
`info` is already of the type `AssemblyName` so there is no need to explicitly create a new name.